modules/cas_cache/ocf_env.c

```diff
@@ -194,6 +204,8 @@ void env_allocator_del(env_allocator *allocator, void *obj)
 			return;
 	}
 
+	/* It wasn't a page from rpool thus need to stop tracking it explicitly */
+	kmemleak_free(obj);
 	kmem_cache_free(allocator->kmem_cache, item);
```

Comment:
shouldn't you do kmemleak_alloc(item) before kmem_cache_free so that this is not detected as double free?